src/flags/experimental.cpp

```diff
@@ -48,8 +47,7 @@ auto const canonicalize_string = [](auto &&rng) {
 
 namespace memgraph::flags {
 
-auto const mapping = std::map{std::pair{"system-replication"sv, Experiments::SYSTEM_REPLICATION},
-                              std::pair{"text-search"sv, Experiments::TEXT_SEARCH},
+auto const mapping = std::map{std::pair{"text-search"sv, Experiments::TEXT_SEARCH},
```

Comment:
This change will make this a breaking change. 
Users who didn't remove "system-replication" from the list will get an error.
I don't think many people used this feature, so just add the `breaking` label to the PR.